webpack.config.js

```diff
@@ -0,0 +1,79 @@
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const BundleTracker = require( 'webpack-bundle-tracker' );
+
+module.exports = {
+  entry: [
+    'webpack-dev-server/client?http://localhost:3000',
+    'webpack/hot/only-dev-server',
+    './app/js/main.js'
```

Comment:
I can't seem to find this `./app/js/main.js` in our repo, we can take it off if we aren't using it. (Else are yet to push it ?)

Reply:
I haven't pushed it yet.  I'm working on converting to 1 entry point.  I can remove it for now, good catch